app/views/admin/users/_member_index.html.erb

```diff
@@ -14,9 +14,28 @@
         </tr>
       </thead>
       <tbody class="crayons-card">
-        <% @users.each do |user| %>
+        <% @users.includes([:organizations]).each do |user| %>
         <tr>
-          <td><%= link_to user.name, admin_user_path(user) %></td>
+          <td>
+          <div class="flex s:items-start flex-col s:flex-row">
+            <a href="<%= admin_user_path(user.id) %>">
+              <img class="radius-full align-middle" src="<%= user.profile_image_url_for(length: 50) %>" width="40" height="40" alt="<%= user.name %>" />
+            </a>
+            <div class="pl-3 flex-1"><%= user.name %>
+              <p class="fs-xs color-base-60">@<%= user.username %>&nbsp;<span class="fs-xs c-indicator" aria-disabled="true"><%= user.current_roles %></span></p>
+            </div>
+          </div>
+          </td>
+          <td><%= user.current_status %></td>
+          <td>
+          <%= user.latest_activity %>
```

Comment:
I noticed that for yesterday we get e.g. `Yesterday Mar, 2022`, when what we want is `Yesterday, 29 Mar, 2022` - maybe a small tweak needed to the logic?